sonar-project.properties

```diff
@@ -0,0 +1 @@
+sonar.projectKey=microwavekonijn_ps2census_AYxTlQvtd4TV1HC_vbtC
```

Comment:
@microwavekonijn sensitive?

Reply:
No, it is just some id. The sensitive stuff is in secrets🙂